subprojects/junit5/junit5.gradle

```diff
@@ -0,0 +1,14 @@
+description = "Mockito preconfigured inline mock maker (intermediate and to be superseeded by automatic usage in a future version)"
```

Comment:
Just so we don't forget: we need to update this description

Reply:
Right, see https://github.com/mockito/mockito/pull/1221#issuecomment-339141751 for a proposal.